workflows/train/deepspeed/cifar/src/train.py

```diff
@@ -187,9 +184,14 @@ def imshow(img):
             loss = running_loss / 2000
             print("[%d, %5d] loss: %.3f" % (epoch + 1, i + 1, loss))
             if args.with_aml_log:
-                mlflow.log_metric("loss", loss)
-            running_loss = 0.0
+                try:
+                    import mlflow
```

Comment:
Non-top-level `import` statements can be confusing. Can you add a comment explaining why we're only importing `mlflow` inside a `try` statement?

Reply:
I had several failed experiments due to mlflow getting broken. Also, some users do not have mlflow in their environment so if they copy/paste this example, we don't want to fail just because mlflow did not work. Does that make sense?

Reply:
@dkmiller - I have added a comment above the import line to explain.